bin/oneoff/back_populate_hoc_survey_prize_purpose

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env ruby
+
+# This script populates the purpose field in Pegasus.hoc_survey_prizes for claimed prizes with no purpose to "Hoc2014"
+
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+
+DB[:hoc_survey_prizes].exclude(claimant: nil).where(purpose: '').update(purpose: 'Hoc2014')
```

Comment:
Fair enough. Though I'll continue to disagree (with existing code). Not having a purpose (until claimed) suggests the code could / would be used for anything other than a specific HOC.